openshift/deployment.yaml

```diff
@@ -48,15 +48,16 @@ objects:
     replicas: 1
     selector:
       matchLabels:
-        app: wps-web
+        app: ${NAME}${SUFFIX}
     template:
       metadata:
         labels:
           app: ${NAME}${SUFFIX}
       spec:
         automountServiceAccountToken: false
         containers:
-          - image: docker-registry.default.svc:5000/auzhsi-tools/wps-web:latest
+          #- image: docker-registry.default.svc:5000/auzhsi-tools/wps-web:latest
```

Comment:
should this still be in here?

Reply:
It will be removed in future changes/commits.